spec/features/sign_in_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+feature 'User sign in', type: :feature do
+  scenario 'correctly login user' do
+    create(:user)
+    visit new_user_session_path
+
+    fill_in 'Email', with: 'user@example.com'
+    fill_in 'Password', with: 'password'
+    select "Public", :from => "statusSelect"
```

Comment:
Do we need this?

Reply:
I don't really know why it's there. It's related to schools. My bad.